app/views/accounts/index.html.erb

```diff
@@ -57,6 +57,8 @@
 
     <p><%= link_to "Neuen Nutzer erstellen", new_account_path, class: 'btn btn-success' %></p>
 
+    <%= render partial: 'search_form', locals: { form_path: accounts_path, search_column: 'email' } %>
```

Comment:
this results in the following mismatching strings:

![Bildschirmfoto 2022-01-12 um 12 24 40](https://user-images.githubusercontent.com/1942953/149131566-84f8ebbb-3b0d-4593-a44d-9ea799793d64.png)

but i think it is not worth the effort to provide a placeholder string in addition to the searchable column, or what do you think?

Reply:
we could solve that using i18n yaml strings

Reply:
best approach. can you adjust it in this scope please?!